hack/make-rules/helm.mk

```diff
@@ -22,7 +22,6 @@ HELM_TAG ?= 0.0.0
 
 CHART_REGISTRY_PATH := oci://${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}
 
-export HELM_EXPERIMENTAL_OCI=1
 export GODEBUG=x509ignoreCN=0
```

Comment:
for versions lower than 3.8 HELM_EXPERIMENTAL_OCI is needed and as `DESIRED_VERSION` can  changed I suggest keeping HELM_EXPERIMENTAL_OCI

Reply:
Why will somebody use the old version?